packages/react-core/src/components/Skeleton/examples/Skeleton.md

```diff
@@ -9,118 +9,25 @@ propComponents: ['Skeleton']
 
 ### Default
 
-```js
-import React from 'react';
-import { Skeleton } from '@patternfly/react-core';
-
-<Skeleton screenreaderText="Loading contents" />;
+```ts file="./SkeletonBasic.tsx"
```

Comment:
For consistency it might be a good idea to rename one of these in order for them to match!